community_tasks/french_evals.py

```diff
@@ -80,6 +80,18 @@ def prompt_gpqa_fr(line, task_name: str = None):
     )
 
 
+# BAC-fr prompt function
+def prompt_bac_fr(line, task_name: str = None):
+    return Doc(
+        task_name=task_name,
+        query=line["prompt"],
+        choices=[""],
+        gold_index=0,
+        instruction="",
+        specific={"instruction": line["instruction"], "enonce": line["enonce"]},
+    )
```

Comment:
```suggestion
    prompt = f"{line['instruction"]}\nEnoncé: {line['question'}"
    return Doc(
        task_name = task_name,
        query = prompt,
        choices = aslist(line["Choix"])
        gold_index = line["Choix"].index(line["Choix correct"])
        instruction = ""
    )
```

Reply:
I would ignore the instruction field for now as it's not task wide (other wise it would be something like "Ceci est une question de mathématiques. Dans le cas où la réponse est une formule, répondre avec la forme la plus simplifiée de l'équation au format latex. Dans le case d'un nombre, répondre avec x précision.")